backend/djangoindia/db/migrations/0005_auto_20240819_0211.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 4.2.5 on 2024-08-18 20:29
+
+import datetime
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def add_created_at_value(apps, schema_editor):
+    contactus = apps.get_model('db', 'contactus')
+    event = apps.get_model('db', 'event')
+    eventregistration = apps.get_model('db', 'eventregistration')
+    newslettersubscription = apps.get_model('db', 'newslettersubscription')
+
+    contactus.objects.update(created_at=django.utils.timezone.now())
+    event.objects.update(created_at=django.utils.timezone.now())
+    eventregistration.objects.update(created_at=django.utils.timezone.now())
+    newslettersubscription.objects.update(created_at=django.utils.timezone.now())
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('db', '0004_alter_contactus_message'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='contactus',
+            name='created_at',
+            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now),
```

Comment:
what is the timezone which `django.utils.timezone.now` returns.
It's always a good practice to keep in UTC.